src/DocumentationBrowserViewExtension/PackageDocumentationManager.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Dynamo.DocumentationBrowser
+{
+    /// <summary>
+    /// Creates a lookup for node annotation markdown files associated with a package.
+    /// Also creates file watchers for the package node annotation paths so new files can be hot reloaded in Dynamo.
+    /// </summary>
+    public class PackageDocumentationManager
```

Comment:
curious why you did not use a static class?

Reply:
hm, yea good question, i guess i had other ideas at the start. Could certainly make this into a static class instead if you prefer that?

Reply:
nope, just curious.